### DISCUSSION TITLE
v7.1.5 Release of PowerShell

### DISCUSSION THREAD

**Discussion by daxian-dbw** [PowerShell/PowerShell#16244] [Show and tell]
### Engine Updates and Fixes

- Handle error from unauthorized access when removing `AppLocker` test files (#15881)
- Test more thoroughly whether a command is `Out-Default` for transcription scenarios (#15653)
- Handle error when the telemetry mutex cannot be created (#15574) (Thanks @gukoff!)
- Configure `ApplicationInsights` to not send cloud role name (Internal 17100)
- Disallow `Add-Type` in NoLanguage mode on a locked down machine (Internal 17522)

### Tools

- Add `.stylecop` to `filetypexml` and format it (#16025)

### Build and Packaging Improvements

<details>

<summary>
<p>Bump .NET SDK to 5.0.402</p>
</summary>

<ul>
<li>Upgrade <code>set-value</code> package for markdown test (#16196)</li>
<li>Sign the .NET <code>createdump</code> executable (#16229)</li>
<li>Move vPack build to 1ES Pool (#16169)</li>
<li>Update to .NET SDK 5.0.402 (Internal 17537)</li>
<li>Move from PkgES hosted agents to 1ES hosted agents (#16023)</li>
<li>Fix the macOS build by updating the pool image name (#16010)</li>
<li>Use Alpine 3.12 for building PowerShell for Alpine Linux (#16008)</li>
</ul>

</details>

### Documentation and Help Content

- Fix example nuget.config (#14349)

[7.1.5]: https://github.com/PowerShell/PowerShell/compare/v7.1.4...v7.1.5

### SHA256 Hashes of the release artifacts

- powershell_7.1.5-1.debian.10_amd64.deb
  - 0B2D70406CBAD653EF4FA9C4A1A8362F99CE8C28D9AFDB29A0A78A67A17953A8
- powershell_7.1.5-1.debian.11_amd64.deb
  - BC90B57B3DEFD23981DBF57FBC3EC78BF990488476621FBF64E63F014C10E73C
- powershell_7.1.5-1.debian.9_amd64.deb
  - 86068A27119400B2ADE46671F3C171C6206BA74C0B1707CF98C5C1AB8F1C67BC
- powershell_7.1.5-1.ubuntu.16.04_amd64.deb
  - A02649B1BADA2A05FE40261AF0311AA4CEF01372CE37D8D158B5F0B598EF1138
- powershell_7.1.5-1.ubuntu.18.04_amd64.deb
  - 79F4DD72BC66ED779DA22C0A561DE0B0CF341201FC970930D9DA9144C2824559
- powershell_7.1.5-1.ubuntu.20.04_amd64.deb
  - 4D0EE19617EBD696527E966A35A00F94CA3E83C850EFDDE10CD4B27369A17288
- powershell-7.1.5-1.centos.8.x86_64.rpm
  - 2D14963EE6DE8A6DE264A2072AADE7496B25FBF0CB69FA7FE186DBC9C80B4886
- powershell-7.1.5-1.rhel.7.x86_64.rpm
  - CE32C979526AFE5CF1A80822BB18965FE77BFDD2B4FF7305425EE6EC645055E7
- powershell-7.1.5-linux-alpine-x64.tar.gz
  - 7793AA7585A2E538FDB1AFB16FB5B08DDE7531E5BAAE1C22C93F0A903B55D151
- powershell-7.1.5-linux-arm32.tar.gz
  - F8BD71230D83EF75865FF4DF08E6A13E31DD0C49A32880F52C334A4B6A1FD894
- powershell-7.1.5-linux-arm64.tar.gz
  - 18EDDD0711591FA1E504B21B1EFFD10DA5813D9B38C4386DA58025BEFDAE2607
- powershell-7.1.5-linux-x64.tar.gz
  - F378DC312CFC07DC96CE23DD55F31714211F0AF4C32640E153AB5B43B467BF7C
- powershell-7.1.5-linux-x64-fxdependent.tar.gz
  - 10D1CE5504E87F8CA574A5276DD3C08C6091A74F9D8657ACD67369B253002ABB
- powershell-7.1.5-osx-x64.pkg
  - 5DC08BE3D40EA2267458797FF3F8370DA4A163C61699DD518F0235C301A2C025
- powershell-7.1.5-osx-x64.tar.gz
  - 0F55F67D48534671FD07017680FE55C48949A748009FEF11B01585C47E6EB398
- PowerShell-7.1.5-win-arm32.zip
  - A37A5E97D693DAC8554B66859D1AB2C4ACC8334898BDFF14E1F3153924484B74
- PowerShell-7.1.5-win-arm64.zip
  - E47FCCAFE2A53AFF2568E536477D93CE4ED7AEE30977B6BF4F7FC38D4A854F14
- PowerShell-7.1.5-win-fxdependent.zip
  - 62500BF2A265691BB55864279DA67B3CAC94066055FDF5C8D6531A6B649D5A75
- PowerShell-7.1.5-win-fxdependentWinDesktop.zip
  - B3835206BEEA59BF155E99A124EEFBBD4EB8469BE1B279F610BD87103308A4EB
- PowerShell-7.1.5-win-x64.msi
  - 17924469091452658680BFBC582BBE5CADE52242F035880CC29D2A763D16D011
- PowerShell-7.1.5-win-x64.zip
  - 6CFF673D859719A6588E5AE9D2B93F940A4FCFF9A37E50C9C0F8286678D1ECAD
- PowerShell-7.1.5-win-x86.msi
  - 6AC379C42444712A3B05B55EAC7AE7CBC625085E79DB344D0012A04A1187268E
- PowerShell-7.1.5-win-x86.zip
  - 35D5C3057E208878FA7F9260270252EA024B3F2B2D35976D9E4BFBB86B788A81

<hr /><em>This discussion was created from the release <a href='https://github.com/PowerShell/PowerShell/releases/tag/v7.1.5'>v7.1.5 Release of PowerShell</a>.</em>

**Comment by yuguuk** [comment]
Is this update going to make it to winget?

**Comment by codev-cory** [comment]
Is that possible to disable the update white tab warning in vscode terminal ？

**Reply by daxian-dbw** [reply]
Yeah, it's can be disabled. Take a look at the help doc: https://docs.microsoft.com/en-us/powershell/module/microsoft.powershell.core/about/about_update_notifications?view=powershell-7.1

**Comment by ipetko96** [comment]
Kinda funny, Defender SmartScreen reporting it as unsafe file.
I am using MS Edge (chromium).

**Reply by daxian-dbw** [reply]
I found the same. Loop in @TravisEz13 and @adityapatwardhan.

**Reply by SteveL-MSFT** [reply]
It seems to have been an issue with Defender Smartscreen and GitHub urls.  We've been working with that team and it appears to have been fixed now.
